res/skins/LateNight/fx_samplers_container_full.xml

```diff
@@ -6,7 +6,7 @@
         <Children>
             <WidgetStack currentpage="[Library],show_big_current" persist="true">
                 <Children>
-                    <WidgetGroup trigger="[Library],unshow_big">
+                    <WidgetGroup trigger="[Master],maximize_library">
```

Comment:
you will probably have to invert all of these with a connection modifier thing because unshow_big means "don't show the maximized library"